content/blog/2020/06/2020-07-07-custom-distribution-service.adoc

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: "Custom Distribution Service : Phase 1 Summary"
+tags:
+- service
+- distribution
+- cloud-native
+- gsoc
+- gsoc2020
+author: sladyn98
+opengraph:
+  image: /images/gsoc/opengraph.png
```

Comment:
Would be better to add a custom image

Reply:
Where do I get this from ?

Reply:
You can create one. E.g. you can take a look at existing ones for inspiration: https://docs.google.com/presentation/d/1Q1PgNnRTgzBpVRXPqQo3PudzCa2eoc6_1_NRjFRMLrU/edit?usp=sharing